Documentation/Tools/HolographicRemoting.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started with Holographic Remoting
+
+Holographic remoting streams holographic content from a PC to your Microsoft HoloLens in real-time, using a Wi-Fi connection. This feature can significantly increase developer productivity when developing mixed reality applications.
+
+## Initial setup
+
+To enable remoting to a HoloLens, it is important to ensure that the project is using the latest remoting components.
+
+1. Open **Window > Package Manager**
+    - Verify that latest version of the **Windows Mixed Reality** package is installed.
+1. Ensure the latest Holographic Remoting application is installed, on the HoloLens, via the Microsoft Store.
```

Comment:
Do we need to instead say "ensure version 2.10 of holographic remoting is installed, per #6811 ?

Reply:
Actually, i was able to successfully use the latest build this morning